setup.cfg

```diff
@@ -31,29 +31,16 @@ include_package_data = True
 packages = find:
 python_requires = >=3.6
 install_requires =
```

Comment:
I think we need to reinstate most of these explicit external pins to protect against future incompatible releases.